Quran/QuranPageCollectionViewCell.swift

```diff
@@ -23,7 +23,7 @@ class QuranPageCollectionViewCell: UICollectionViewCell {
     @IBOutlet weak var scrollView: UIScrollView!
 
     var page: QuranPage?
-
+    var ayahInfoData: [AyahNumber: [AyahInfo]]?
```

Comment:
We don't need `ayahInfoData`, since it is already in the `HighlightingView` view and we don't need it here at all.